content/docs/command-reference/checkout.md

```diff
@@ -187,16 +189,10 @@ outs:
     md5: 98af33933679a75c2a51b953d3ab50aa
 ```
 
-But if you check `model.pkl`, the file hash is still the same:
-
-```dvc
-$ md5 model.pkl
-MD5 (model.pkl) = ab349c2b5fa2a0f66d6f33f94424aebe
-```
-
-This is because `git checkout` changed `dvc.lock` and other DVC files. But it
-did nothing with the `model.pkl` and `matrix.pkl` files. Git doesn't track those
-files; DVC does, so we must do this:
+But if you check the MD5 of `model.pkl`, the file hash is still the same
```

Comment:
why don't we mention matrix file? it's clear that checkout deals with multiple files this way

Reply:
Matrix file?

Reply:
matrix.pkl mentioned above

Reply:
> checkout deals with multiple files

Updated to mention this directly instead in 508cf979.